framework/src/bundle/CoreBundleContext.h

```diff
@@ -197,6 +202,7 @@ namespace cppmicroservices
         // The core context is exclusively constructed by the FrameworkFactory class
         friend class FrameworkFactory;
 
+        bool stopped;
```

Comment:
can this be a `atomic_bool`? Given how this variable is used, it also removes the need to lock a mutex in `Stop/StartFramework` and `FrameworkStopped`.

Reply:
Fixed